v4/source/system.yaml

```diff
@@ -84,8 +91,17 @@
             Client := model.NewAPIv4Client("https://your-mattermost-url.com")
             Client.Login("email@domain.com", "Password1")
 
+            config := model.Config{
+                EmailSettings: model.EmailSettings{
```

Comment:
Just a minor comment: Indentation is not aligned. Might need to reduce by 2 spaces.

Reply:
done